### PR TITLE
[1.9] fix #2768: When CHROM / POS columns are hidden and you click a geneti…

### LINKED ISSUE #2768: When CHROM / POS columns are hidden and you click a genetic data row, you get "viewFeatures called with silly scale"
Happened right now after deleting TypeTest entity on molgenis62, when data explorer responded by returning to testSamples_KEGG entity.

Also seen it during the automated annotator test.

No clear way to reproduce it yet.

### PR DESCRIPTION
**Pull request by bartcharbon**
…c data row, you get "viewFeatures called with silly scale"

### CODE REVIEW

File: molgenis-dataexplorer/src/main/resources/js/dataexplorer-data.js [hunk 1]
```diff
@@ -148,6 +148,12 @@
 		return molgenis.ie9 !== true && genomebrowserStartAttribute !== undefined && genomebrowserChromosomeAttribute !== undefined && molgenis.dataexplorer.settings["data_genome_browser"] !== false;
 	}
 
+	//used to determine if the rowclick should be available
+	function isGenomeBrowserAttributesSelected(){
+    		var attributes = molgenis.dataexplorer.getSelectedAttributes();
+            return (attributes.indexOf(genomebrowserChromosomeAttribute)!=-1) && (attributes.indexOf(genomebrowserStartAttribute)!=-1);
+    }
+
```
Comment (fdlk) [inline molgenis-dataexplorer/src/main/resources/js/dataexplorer-data.js]: cunning! :)

**Comment by fdlk** [comment]
But still broken I fear.

**Comment by fdlk** [comment]
Can one of the admins verify this patch?

**Comment by fdlk** [comment]
add to whitelist

**Comment by bartcharbon** [comment]
Seen it work locally, rowclick should get disabled when you deselect one of the needed attributes(chrom or pos)... 
can you describe what you did to reproduce the error still?

**Comment by bartcharbon** [comment]
reviewing the solution I think the fix does not cover the situation where you start out without the POS and CHROM attributes selected. The solution "fires" on the change of the attributeselection, can this be the case for your test?

**Comment by fdlk** [comment]
that would make sense, yes. I think I reload the page with the chrom/pos attributes deselected

**Comment by bartcharbon** [comment]
updated the fix: added same code to the initial properties for the table

**Pull request merged**